zipline/algorithm.py

```diff
@@ -138,44 +138,74 @@ def handle_data(context, data):
-               Whether to fill orders immediately or on next bar.
-            asset_finder : An AssetFinder object
-                A new AssetFinder object to be used in this TradingEnvironment
-            equities_metadata : can be either:
-                            - dict
-                            - pandas.DataFrame
-                            - object with 'read' property
-                If dict is provided, it must have the following structure:
-                * keys are the identifiers
-                * values are dicts containing the metadata, with the metadata
-                  field name as the key
-                If pandas.DataFrame is provided, it must have the
-                following structure:
-                * column names must be the metadata fields
-                * index must be the different asset identifiers
-                * array contents should be the metadata value
-                If an object with a 'read' property is provided, 'read' must
-                return rows containing at least one of 'sid' or 'symbol' along
-                with the other metadata fields.
-            identifiers : List
-                Any asset identifiers that are not provided in the
-                equities_metadata, but will be traded by this TradingAlgorithm
+        Parameters
+        ----------
+        *args
+            Forwarded ``initialize``.
+        initialize : callable[context -> None, optional
+            Function that is called at the start of the simulation to
+            setup the initial context.
+        handle_data : callable[(context, data) -> None], optional
+            Function called on every bar. This is where most logic should be
+            implemented.
+        before_trading_start : callable[(context, data) -> None], optional
+            Function that is called before any bars have been processed each
+            day.
+        analyze : callable[(context, DataFrame) -> None], optional
+            Function that is called at the end of the backtest. This is passed
+            the context and the performance results for the backtest.
+        script : str, optional
+            Algoscript that contains the definitions for the four API
+            functions and any supporting code.
+        namespace : dict, optional
+            The namespace to execute the algoscript in. By default this is an
+            empty namespace that will include only python built ins.
+        algo_filename : str, optional
+            The filename for the algoscript. This will be used in exception
+            tracebacks. default: '<string>'.
+        data_frequency : {'daily', 'minute'}, optional
+            The duration of the bars.
+        capital_base : float, optional
+            How much capital to start with. default: 1.0e5
+        instant_fill : bool, optional
+            Whether to fill orders immediately or on next bar. default: False
+        asset_finder : AssetFinder, optional
```

Comment:
Thanks, I didn't audit the existing entries. I will make sure there are no other invalid params there.